guides/source/documents.yaml

```
@@ -11,39 +11,65 @@
       url: active_record_basics.html
-      description: This guide will get you started with models, persistence to database, and the Active Record pattern and library.
+      description: >
+        Active Record allows your models to interact with the application's database.
+        This guide will get you started with
+        Active Record models and persistence to the database.
     -
       name: Active Record Migrations
       url: active_record_migrations.html
-      description: This guide covers how you can use Active Record migrations to alter your database in a structured and organized manner.
+      description: >
+        Migrations are a feature of Active Record that allows you to evolve your
+        database schema over time. Rather than write schema modifications in pure SQL,
+        migrations allow you to use a Ruby DSL to describe changes to your tables.
     -
       name: Active Record Validations
       url: active_record_validations.html
-      description: This guide covers how you can use Active Record validations.
+      description: >
+        Validations are used to ensure that only valid data is saved into your
+        database. This guide teaches you how to validate the state of objects
+        before they go into the database using Active Record's validations
+        feature.
     -
       name: Active Record Callbacks
       url: active_record_callbacks.html
-      description: This guide covers how you can use Active Record callbacks.
+      description: >
+        With callbacks it is possible to write code that will run whenever an
+        object is created, updated, destroyed, etc. This guide teaches you how
+        to hook into this object life cycle of your Active Record objects.
     -
       name: Active Record Associations
       url: association_basics.html
-      description: This guide covers all the associations provided by Active Record.
+      description: >
+        In Active Record, an association is a connection between two Active
+        Record models. This guide covers all the associations provided by
+        Active Record.
     -
       name: Active Record Query Interface
       url: active_record_querying.html
-      description: This guide covers the database query interface provided by Active Record.
+      description: >
+        Instead of using raw SQL to find database records, Active Record
+        provides better ways to carry out the same operations. This guide
+        covers different ways to retrieve data from the database using Active
+        Record.
     -
       name: Active Model Basics
       url: active_model_basics.html
-      description: This guide covers the use of model classes without Active Record.
+      description: >
+        Active Model allows for Action Pack helpers to interact with plain Ruby
+        objects. Active Model also helps build custom ORMs for use outside of
+        the Rails framework. This guide should provide you with all you need
```

Perhaps we can be a bit more assertive here? :)

```suggestion
        the Rails framework. This guide provides you with all you need
```

---

Yes :)